### ISSUE TITLE
Find out how to create tests for a CLI and which testing library to use

### ISSUE THREAD

**Issue by AlexGaiser** [AlexGaiser/fast-react#7]
I personally am a big fan of Jest, and think it will have all the tools we need for this project. 

Once we have testing in place we will need to add branch protections.

**Comment by AlexGaiser** [comment]
@brian-ogilvie Were there any additional libraries you found for testing as a CLI tool?

**Comment by brian-ogilvie** [comment]
Are we interested in using `react-testing-library` to verify that our generated app can load and run properly?

**Comment by AlexGaiser** [comment]
@brian-ogilvie Yes we are, and I wanted to let you know that I think we will need `ts-jest` as a dev dependency

**Comment by brian-ogilvie** [comment]
I think `@types/jest` accomplishes what we need.
